crates/lunatic-distributed/Cargo.toml

```diff
@@ -18,7 +18,7 @@ bytes = "1"
 dashmap = { workspace = true }
 log = { workspace = true }
 quinn = { version = "0.9" }
-rcgen = { version = "0.10", features = ["pem", "x509-parser"] }
+rcgen = { workspace = true }
```

Comment:
We had a discussion about which crates should be workspace defined. Maybe rcgen shoudn't as it's used only in distributed and for PEM format shouln't matter which version/lib generated it.